packages/clerk-js/src/ui/elements/Section.tsx

```diff
@@ -247,6 +247,10 @@ export const SectionHeader = (props: SectionHeaderProps) => {
         variant='h3'
         elementDescriptor={textElementDescriptor}
         elementId={textElementId}
+        sx={t => ({
+          fontWeight: t.fontWeights.$medium,
```

Comment:
We don't use directly fontWeights. Text should be represented only by the text variants we already have in our typography.

Reply:
Fixed!